mubench.pipeline/tasks/implementations/compile.py

```diff
@@ -179,7 +179,7 @@ def __parse_maven_classpath(shell_output: str) -> List[str]:
         return classpath
 
     @staticmethod
-    def __parse_ant_classpath(shell_output: str) -> List[str]:
+    def __parse_ant_classpath(shell_output: str) -> Set[str]:
```

Comment:
What's the effect of this change? This is only a type int, isn't it? It doesn't change the type of the result, i.e., these methods still return lists with potential duplicates.